model-client/src/commonMain/kotlin/org/modelix/model/client2/IModelClientV2.kt

```diff
@@ -86,4 +86,8 @@ interface IModelClientV2 {
     suspend fun <R> query(branch: BranchReference, body: (IMonoStep<INode>) -> IMonoStep<R>): R
 
     suspend fun <R> query(repositoryId: RepositoryId, versionHash: String, body: (IMonoStep<INode>) -> IMonoStep<R>): R
+
+    suspend fun getObjects(repository: RepositoryId, keys: Sequence<String>): Map<String, String>
+
+    suspend fun pushObjects(repository: RepositoryId, objects: Sequence<Pair<String, String>>)
```

Comment:
These lack documentation. Especially, what is the `Pair<String, String>>`? Explanatory data class instead?

Reply:
Introduced some typealiases.